compose/compose-file/index.md

```diff
@@ -882,6 +882,81 @@ port (a random host port will be chosen).
      - "127.0.0.1:5000-5010:5000-5010"
      - "6060:6060/udp"
 
+### secrets
+
+After you
+[define or reference secrets in the top-level `secrets` configuration](#secrets-configuration-reference),
+grant access to secrets on a per-service basis using the per-service `secrets`
+configuration. Two different syntax variants are supported.
```

Comment:
>  the compose file reference is literally the only source for a lot of this stuff.

That's not really an argument for confusing the type of documentation, it's just an argument for including some more docs about it somewhere else (assuming we need it, which I'm not sure we do).

>  "secrets" can't allow you to do anything. "secrets" are not an actor.

Ok, we can be more verbose. "The `secrets` field allows you..."

> Can you unpick why this bothers you?

I think it's because I prefer to read reference docs and it feels more comfortable to me to read things written in the passive voice.

Wouldn't it be good to remain consistent with the rest of the document?